sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -408,7 +407,7 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
   async peekMessagesBySession(
     sessionId: string,
     messageCount: number,
-    options?: OperationOptionsBase & SendManagementRequestOptions
+    options: SendManagementRequestOptions = {}
```

Comment:
Same here, doesn't need to be defaulted.

Reply:
Done.